src/sentry/snuba/metrics/extraction.py

```diff
@@ -250,6 +252,10 @@ def _get_aggregate_fields(aggregate: str) -> Sequence[str]:
     return []
 
 
+def is_standard_metrics_query(query: Optional[str] = "") -> bool:
```

Comment:
Could we expose just one out of `is_on_demand_query` and this? This is a misleading function for a public interface, because in order to check for a compatible query you _always_ have to check for both the selected aggregates _AND_ the query (due to countIf).